sources/packages/api/src/types/formsFlow.ts

```diff
@@ -3,3 +3,8 @@ export interface CreateApplicationPayload {
   formId: string;
   submissionId: string;
 }
+
+export enum checkboxFormioType {
```

Comment:
Do we need to have a type to have this explicit?
If yes we need to create the enum following the conventions on the wiki.
Besides that, the name is too specific (IMO) and we can have it at the service level maybe with a generic name like YesNoOptions or similar.

Reply:
I liked the idea of creating an enum for the yes or no option, that's why I created this, let me know @andrewsignori-aot  @guru-aot  your thoughts? 
yup suggested like sounds generic.

Reply:
removed

Reply:
Yes we have have a generic options enum which derives the yes or no options.